.travis.yml

```diff
@@ -4,18 +4,19 @@ python:
 addons:
   postgresql: "9.4"
 before_install:
-  - sudo apt-get update
   - pip install coverage
   - pip install coveralls
 install:
   - pip install -r requirements/dev.txt
 before_script:
-  - sudo createdb -U postgres openstax
-  - ./manage.py migrate
+  - createdb -U postgres openstax
+  - python manage.py migrate
 script:
-  - ./manage.py test
+  # This is the same as  'python manage.py test demo' with a coverage wrapper
```

Comment:
Might want to remove the reference to demo in the comment here to keep confusion to a minimum.